store-implementation/accumulo-store/README.md

```diff
@@ -57,7 +57,7 @@ Accumulo set up
 
 Gaffer has been extensively tested with Accumulo version 1.7.2. It should work with any of the 1.7.* versions of Accumulo.
 
-For the purposes of unit testing and very small-scale ephemeral examples, Gaffer offers a [MockAccumuloStore](accumulo-store/src/main/java/gaffer/accumulostore/MockAccumuloStore.java). This uses Accumulo's `MockInstance` to create an in-memory Accumulo store that runs within the same JVM as the client code. All data in this store will disappear when the JVM is shut down.
+For the purposes of unit testing and very small-scale ephemeral examples, Gaffer offers a [MockAccumuloStore](accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/MockAccumuloStore.java). This uses Accumulo's `MockInstance` to create an in-memory Accumulo store that runs within the same JVM as the client code. All data in this store will disappear when the JVM is shut down.
```

Comment:
MockAccumuloStore link should be:

src/main/java/uk/gov/gchq/gaffer/accumulostore/MockAccumuloStore.java